src/index.css

```diff
@@ -3,3 +3,23 @@ body {
   padding: 0;
   font-family: sans-serif;
 }
+
+.profile-form form {
```

Comment:
Any reason why you chose to put some styles in `index.css` and some directly in the React component?

Reply:
For the form, I couldn't figure out a way to add transparency to the backgroundColor property.  I don't think there was any reasoning behind the others.  I could put them all in the React component if that's the best style.